### ISSUE TITLE
Error creating new project

### ISSUE THREAD

**Issue by Jin35** [flutter/flutter-intellij#3399]
## What happened

Click on "Create new flutter project" - no wizard shown, just crush report appears

## Version information

Android Studio `3.5 Canary 12` • Flutter plugin `io.flutter 34.0.4` • Dart plugin `191.6183.88`

Error getting Flutter sdk information.

- from AndroidStudio settings:
Flutter 1.5.4 • channel dev • https://github.com/flutter/flutter.git Framework • revision b593f5167b (23 hours ago) • 2019-04-22 07:51:33 -0700 Engine • revision ca31a7c57b Tools • Dart 2.3.0 (build 2.3.0-dev.0.1 cf4444b803)

## Exception

null

```
java.lang.NoSuchMethodError: com.android.tools.idea.observable.ListenerManager.receive(Lcom/android/tools/idea/observable/ObservableValue;Lcom/android/tools/idea/observable/Receiver;)V
	at io.flutter.project.FlutterSettingsStep.<init>(FlutterSettingsStep.java:68)
	at io.flutter.project.FlutterProjectStep.createDependentSteps(FlutterProjectStep.java:346)
	at com.android.tools.idea.wizard.model.ModelWizard.addStep(ModelWizard.java:225)
	at com.android.tools.idea.wizard.model.ModelWizard.addStep(ModelWizard.java:227)
	at com.android.tools.idea.wizard.model.ModelWizard.<init>(ModelWizard.java:93)
	at com.android.tools.idea.wizard.model.ModelWizard.<init>(ModelWizard.java:47)
	at com.android.tools.idea.wizard.model.ModelWizard$Builder.build(ModelWizard.java:558)
	at io.flutter.actions.FlutterNewProjectAction.actionPerformed(FlutterNewProjectAction.java:48)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:260)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:277)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.lambda$actionPerformed$0(ActionMenuItem.java:292)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:283)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:107)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:282)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$fireActionPerformed$0(ActionMenuItem.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.runSyncTransaction(TransactionGuardImpl.java:88)
	at com.intellij.openapi.application.TransactionGuardImpl.lambda$submitTransaction$1(TransactionGuardImpl.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:120)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:121)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:111)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:524)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.access$300(BegMenuItemUI.java:35)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:546)
	at java.awt.Component.processMouseEvent(Component.java:6550)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3325)
	at java.awt.Component.processEvent(Component.java:6315)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4899)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4535)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:766)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:739)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:736)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:747)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:692)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

## flutter doctor -v
[√] Flutter (Channel dev, v1.5.4, on Microsoft Windows [Version 10.0.17763.253], locale en-US)
    • Flutter version 1.5.4 at C:\work\flutter
    • Framework revision b593f5167b (23 hours ago), 2019-04-22 07:51:33 -0700
    • Engine revision ca31a7c57b
    • Dart version 2.3.0 (build 2.3.0-dev.0.1 cf4444b803)

[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\Evgenii_Grigoriev\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: C:\work\android-studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b01)
    • All Android licenses accepted.

[√] Android Studio (version 3.4)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 34.0.2
    • Dart plugin version 183.5901
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[√] Android Studio (version 3.5)
    • Android Studio at C:\work\android-studio
    • Flutter plugin version 34.0.4
    • Dart plugin version 191.6183.88
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b01)

[√] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)

• No issues found!

**Comment by devoncarew** [comment]
This looks like an issue with Android Studio `3.5 Canary 12`. @Jin35, thanks for the report. We don't support canary versions, but will need to adopt to this change so that we can work with the upcoming beta.

```
java.lang.NoSuchMethodError: com.android.tools.idea.observable.ListenerManager.receive(Lcom/android/tools/idea/observable/ObservableValue;Lcom/android/tools/idea/observable/Receiver;)V
```

cc @stevemessick

**Comment by stevemessick** [comment]
Fixed in #3380
